desktop/assets/ruffle_desktop.rc

```diff
@@ -1,4 +1,5 @@
 // winres.h cannot be included, so define manually
+#define VS_VERSION_INFO 1
```

Comment:
I'm wondering about these version-related strings in this file (`VER_FILEVERSION`, `VER_PRODUCTVERSION`). Are they important? If they are, we should probably set them dynamically based on the version from cargo somehow?

Reply:
Yes we should! They're not critical by any means, but it's good practice to set them. It's the windows equivalent of tools giving `--version` as a standard CLI argument.

Reply:
I guess we could set them dynamically in `build.rs` then. It seems that we can use the `macros` parameter for that, however I'm unsure how that works and I wasn't able to find any examples. We should also consider parameterizing `VER_DEBUG` and possibly the year.